src/libraries/System.Private.CoreLib/src/System/Text/Rune.cs

```diff
@@ -102,7 +107,9 @@ public Rune(int value)
         /// <exception cref="ArgumentOutOfRangeException">
         /// If <paramref name="value"/> does not represent a value Unicode scalar value.
         /// </exception>
+#if SYSTEM_PRIVATE_CORELIB // CS3019: CLS compliance checking will not be performed on 'Rune.explicit operator Rune(uint)' because it is not visible from outside this assembly
```

Comment:
Rather than inserting all those conditionals, couldn't we just conditionally suppress the diagnostic at the top of the source file?